src/pages/People/People.tsx

```diff
@@ -0,0 +1,125 @@
+import { PeopleFilters } from '../../components/PeopleFilters/PeopleFilters';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { useSearchParams } from 'react-router-dom';
+import { filter } from '../../function/filter';
+import { FILTER } from '../../types/filters';
+
+export const People = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState<boolean>(false);
+
+  const filterSex = searchParams.get('sex') as FILTER || FILTER.ALL;
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
The retrieval of search parameters for `sex`, `query`, and `centuries` is correct. Ensure that these parameters are correctly set and updated in the URL to maintain consistent filtering behavior.